flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/util/FileSpout.java

```diff
@@ -38,6 +38,8 @@
 	protected String path = null;
 	protected BufferedReader reader;
 
+	protected boolean finished;
+
 	public FileSpout() {}
```

Comment:
Yes. But as the argument from above explains. All spout/bolts should be written in a Flink agnostic way. No Storm developer would write it like this, because this flag does only make sense if you know about `FinitSpout` interface -- but you should not assume to know about it.